propagation-stackdriver/src/main/java/zipkin2/propagation/stackdriver/CompositeExtractor.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2016-2018 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.propagation.stackdriver;
+
+import brave.propagation.TraceContext;
+import brave.propagation.TraceContextOrSamplingFlags;
+
+public class CompositeExtractor<C> implements TraceContext.Extractor<C> {
+
+	private TraceContext.Extractor<C>[] extractors;
+
+	public CompositeExtractor(TraceContext.Extractor<C>... extractors) {
+		this.extractors = extractors;
+	}
+
```

Comment:
for this to work effectively it needs to be a propagation factory. any member having supportsJoin false makes the whole thing supports join false. similarly on traceid128

Reply:
I'm not sure if I understand 100%, but is this accomplished by the `supportsJoin()` returning `false` and `requires128BitTraceId()` returning true in `StackdriverTracePropagation.FACTORY`?

I don't understand how another propagation would fit here, and would be used in conjunction with `StackdriverTracePropagation`, but I might be missing something.